README.md

```diff
@@ -295,3 +295,30 @@ spec:
     kind: VolumeSnapshot
     name: my-cool-snap-1
 ```
+
+## Multi-cluster
```

Comment:
should we make it clearer that only GKE is supported? Maybe create a subsection for GKE or rename this section to reflect it.